packages/playwright-test/types/testReporter.d.ts

```diff
@@ -493,6 +493,7 @@ export interface JSONReportTestResult {
   stderr: JSONReportSTDIOEntry[];
   retry: number;
   steps?: JSONReportTestStep[];
+  startTime?: Date;
```

Comment:
Btw, can we make it required here or is it optional on the result itself?

Reply:
I believe we can make it required, I left it optional because I didn't want this change to break anyone's reporter if they used those types.

Reply:
This should not break the call sites since clients are the consumer of this api. You can add options and change optional to required, but not the other way.

Reply:
So I'm suggesting that you make this change and convert `startTime` into a required field. Our main public API is `testReporter.d.ts`, and it is already required there. What you are editing here is a JSON-serialized format, which is still public, but much less important. If we managed to commit to this field being in test reporter API, committing to it in JSON is an easy sell.

Reply:
I see, I'll make that change